### ISSUE TITLE
Drawer not able to start opened within MatTab

### ISSUE THREAD

**Issue by Bored-Bohr** [angular/material2#11154]
#### Bug, feature request, or proposal:
When a drawer with attribute `opened="true"` is put within a tab it fails to initialize as opened correctly.

#### What is the expected behavior?
When a drawer with attribute `opened="true" is put within a tab, it should be opened during the first time you change to that tab.

#### What is the current behavior?
When a drawer with attribute `opened="true" is put within a tab, it is closed during the first time you change to that tab.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-lmsb5b?file=app%2Fdrawer-in-tab-example.ts

#### What is the use-case or motivation for changing an existing behavior?
The drawer is currently starting closed and if toggled animates from opened to close. It should follow expected behavior of starting open.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular CLI: 1.7.4
Node: 8.9.0
OS: win32 x64
Angular: 5.2.10
... animations, common, compiler, compiler-cli, core, forms
... http, platform-browser, platform-browser-dynamic
... platform-server, router

@angular/cdk: 5.2.5
@angular/cli: 1.7.4
@angular/material: 5.2.5
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.2
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.8.3
webpack: 3.11.0

#### Is there anything else we should know?
The drawer does not properly set 
drawer:
```
transform: translate3d(0px, 0px, 0px);
visibility: visible;
```
drawer-content for `mode="side"`:
```
margin-left: <drawer-width>px;
margin-right: 0px;
```

**Comment by josephperrott** [comment]
Closing as this issue was actually addressed in our 6.0 release.  For components that use animations to set their initial state, such as MatDrawer, you can use lazy loading tabs.

I [forked your stackblitz](https://stackblitz.com/edit/angular-lmsb5b-tyulrt?file=app/drawer-in-tab-example.html) and updated to 6.0 to show how this can be done.

**Comment by Bored-Bohr** [comment]
Thanks, I didn't realize that this was related to that issue.
